### ISSUE TITLE
BibleTime crashes with Qt 5.6 in release mode on MacOS

### ISSUE THREAD

**Issue by mgruner** [bibletime/bibletime#58]
If I build BibleTime master (2.11 beta1) in release mode with Qt 5.6 (installed from homebrew), it crashes on startup: http://pastebin.com/fRKv4d35

The following external images are referenced from the BibleTime executable:
```MacOS/BibleTime:
	@executable_path/../MacOS/libclucene-core.1.dylib (compatibility version 1.0.0, current version 2.3.3)
	@executable_path/../MacOS/libclucene-shared.1.dylib (compatibility version 1.0.0, current version 2.3.3)
	@executable_path/../MacOS/libsword-1.7.4.dylib (compatibility version 0.0.0, current version 0.0.0)
	@executable_path/../Frameworks/QtWebEngine.framework/Versions/5/QtWebEngine (compatibility version 5.6.0, current version 5.6.0)
	@executable_path/../Frameworks/QtWebEngineWidgets.framework/Versions/5/QtWebEngineWidgets (compatibility version 5.6.0, current version 5.6.0)
	@executable_path/../Frameworks/QtXml.framework/Versions/5/QtXml (compatibility version 5.6.0, current version 5.6.0)
	@executable_path/../Frameworks/QtPrintSupport.framework/Versions/5/QtPrintSupport (compatibility version 5.6.0, current version 5.6.0)
	@executable_path/../Frameworks/QtSvg.framework/Versions/5/QtSvg (compatibility version 5.6.0, current version 5.6.0)
	@executable_path/../Frameworks/QtWebEngineCore.framework/Versions/5/QtWebEngineCore (compatibility version 5.6.0, current version 5.6.0)
	@executable_path/../Frameworks/QtWebChannel.framework/Versions/5/QtWebChannel (compatibility version 5.6.0, current version 5.6.0)
	@executable_path/../Frameworks/QtPositioning.framework/Versions/5/QtPositioning (compatibility version 5.6.0, current version 5.6.0)
	@executable_path/../Frameworks/QtQuick.framework/Versions/5/QtQuick (compatibility version 5.6.0, current version 5.6.0)
	@executable_path/../Frameworks/QtQml.framework/Versions/5/QtQml (compatibility version 5.6.0, current version 5.6.0)
	@executable_path/../Frameworks/QtNetwork.framework/Versions/5/QtNetwork (compatibility version 5.6.0, current version 5.6.0)
	@executable_path/../Frameworks/QtWidgets.framework/Versions/5/QtWidgets (compatibility version 5.6.0, current version 5.6.0)
	@executable_path/../Frameworks/QtGui.framework/Versions/5/QtGui (compatibility version 5.6.0, current version 5.6.0)```

I'll gladly provide more information if needed. Is there a problem in the Qt 5.6 build of homebrew, or possibly in our build toolchain?

I built the binary package for 2.11 beta1 for MacOS with Qt 5.5.1, and that worked fine.

Thanks for all help!

**Comment by jaakristioja** [comment]
Can't open the pastebin.com link you provided.

**Comment by gholmlund** [comment]
In the pastbin I see that the call stack included our "MyMessageOutput". This should output a message to standard error or a log file. I am not familiar with which it is on a Mac. This message is what we will need to troubleshoot the problem. I suspect it is a missing Qt plugin or Qt shared library.

**Comment by mgruner** [comment]
```(BibleTime 2.11.0_beta1+7b36a6b58390ae87e0beabb0f76b06f86b237919) FATAL: Could not find QtWebEngineProcess
Please report this bug at https://github.com/bibletime/bibletime/issues
Abort trap: 6```

**Comment by mgruner** [comment]
https://bugreports.qt.io/browse/QTBUG-50155?

**Comment by mgruner** [comment]
I currently don't see a QWebEngineProcess in theg generated bundle at all. Do we have to add this to the cmake scripts manually?

```find . -name "*WebEngine*"
./Contents/Frameworks/QtWebEngine.framework
./Contents/Frameworks/QtWebEngine.framework/QtWebEngine
./Contents/Frameworks/QtWebEngine.framework/Versions/5/QtWebEngine
./Contents/Frameworks/QtWebEngineCore.framework
./Contents/Frameworks/QtWebEngineCore.framework/QtWebEngineCore
./Contents/Frameworks/QtWebEngineCore.framework/Versions/5/QtWebEngineCore
./Contents/Frameworks/QtWebEngineWidgets.framework
./Contents/Frameworks/QtWebEngineWidgets.framework/QtWebEngineWidgets
./Contents/Frameworks/QtWebEngineWidgets.framework/Versions/5/QtWebEngineWidgets```

**Comment by mgruner** [comment]
I believe I was able to fix this in a9831df.
